decidim-elections/app/views/decidim/elections/admin/steps/index.html.erb

```diff
@@ -1,3 +1,4 @@
+<% add_decidim_page_title(t("title", scope: "decidim.elections.trustee_zone.elections.key_ceremony_steps", election: translated_attribute(election.title))) %>
```

Comment:
This title is wrong because the one from `.item_show__header-title` is also wrong. For the moment lets ignore it as we should focus on this page with attention some other time. 

At least is consistent 😄